tests/HorribleDeprecationWorkarounds.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use PHPUnit\Framework\Assert;
+
+/**
+ * @author    Craig Gosman <craig@ingenerator.com>
+ * @licence   proprietary
+ *
+ *
+ * This is hideous so is self contained in a file and uses traits so we can find them and refactor in the future
+ */
+trait Workarounds
```

Comment:
Could we call this something specific - `ObjectInternalAccessTestWorkarounds` or something? Just so we remember what it is and so we're not tempted to add other types of workaround to it in future?

Reply:
I will. It wont stop us adding more traits under different names to workaround things in future though ;)
Haven't written the changelog yet was waiting to see if we're going for a point release on this. What do you reckon re phpunit? Is it worth a look at incase I'm just missing something obvious or will we ignore it for now?

Reply:
Absolutely, that was what I was meaning :) Best to keep the different types of unpleasant workarounds separate.

I think don't worry about phpunit 9 just now, we can revisit later. There's no security implications to the EOL and if it passes on 8 now it should continue to.

Reply:
Grand I'll release it now and it might make the dep build at 9